pulp_ansible/app/viewsets.py

```diff
@@ -508,6 +809,44 @@ class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet):
 
     parent_viewset = AnsibleRepositoryViewSet
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repository_model_or_obj_perms:ansible.view_ansiblerepository",
+            },
+            {
+                "action": "destroy",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:ansible.delete_ansiblerepository",
+                    "has_repository_model_or_obj_perms:ansible.view_ansiblerepository",
+                ],
+            },
+            {
+                "action": ["repair"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:ansible.repair_ansiblerepository",
+                    "has_repository_model_or_obj_perms:ansible.view_ansiblerepository",
+                ],
+            },
+            {
+                "action": ["rebuild_metadata"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:ansible.rebuild_metadata_ansiblerepository",
+                ],
+            },
+        ],
+        "queryset_scoping": {"function": "scope_queryset"},
```

Comment:
I think `queryset_scoping` should be removed. This endpoint is already scoped by permission on the parent repository. Don't need to filter it any further.